TelAviv/TAfait006.xml

```diff
@@ -33,13 +33,13 @@
                             Fol. 38a. Falasha prayers. All these prayers are published by Aescoly un the book mentioned above, out of a manuscript in the collection d'Abbadie. 38a corresponds to Cantique Aescoly p. 152 -- 39b corresponds to Hymne Aescoly p. 156 -- 41a corresponds to Psaume et Prière Aescoly p. 158 -- 48b. corresponds to Prière magique Aescoly p. 176 -- 56a. corresponds to Prière magique Aescoly p. 186</summary>
```

Comment:
Avoid "falasha" also in line 33.

Reply:
This is a "given summary" imported as is.